### DISCUSSION TITLE
From PDF to text

### DISCUSSION THREAD

**Discussion by BiEchi** [PaddlePaddle/PaddleOCR#12234] [Q&A] [answered]
Greetings! I'm trying to convert PDF to plain text instead of Word format. Is there any way PaddleOCR and do it? Looking forward to your reply!

**Comment by hungtrieu07** [comment, marked as answer]
You should convert PDF file to images file, and push that images into PaddleOCR with your language. If your language don't support, then need to train a recognition model.
